person.rb

```diff
@@ -0,0 +1,60 @@
+require './nameable'
+require './rental'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
```

Comment:
- [ ] In order to set up the `has-many` relationship between the `Person` and `Rentals`, we need:
1. Getters and Setters for `@rentals` tp be present e.g `attr_accessor :rentals`
2. `@rentals` array should be initialized in the constructor